kubernetes-filebeat/templates/configmap.yaml

```diff
@@ -0,0 +1,62 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "kubernetes-filebeat.fullname" . }}-config
+  labels:
+    {{- if .Values.cluster_service }}
+    k8s-app: {{ template "kubernetes-filebeat.fullname" . }}
+    kubernetes.io/cluster-service: "true"
+    {{- end }}
+    app: {{ template "kubernetes-filebeat.name" . }}
+    chart: {{ template "kubernetes-filebeat.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+data:
+  filebeat.yml: |-
+    filebeat.config:
+      prospectors:
+        # Mounted `filebeat-prospectors` configmap:
+        path: ${path.config}/prospectors.d/*.yml
+        # Reload prospectors configs as they change:
+        reload.enabled: false
+      modules:
+        path: ${path.config}/modules.d/*.yml
+        # Reload module configs as they change:
+        reload.enabled: false
+    processors:
+      - add_cloud_metadata:
+
+    {{- if .Values.app.elastic_cloud_enabled }}
```

Comment:
We don't use elastic-cloud and we can't test this configuration. I think we can drop this part of this configuration to improve the readability.